compass/ocean/tests/global_ocean/init/__init__.py

```diff
@@ -46,22 +47,25 @@ def __init__(self, test_group, mesh, initial_condition, with_bgc):
         """
         name = 'init'
         mesh_name = mesh.mesh_name
+        ic_dir = initial_condition
         if with_bgc:
-            ic_dir = '{}_BGC'.format(initial_condition)
-        else:
-            ic_dir = initial_condition
+            ic_dir = '{}_BGC'.format(ic_dir)
+        if with_inactive_top_cells:
+            ic_dir = '{}_inactive_top'.format(ic_dir)
```

Comment:
Here, it will also be useful to store the subdirectory for the "original" test case we want to compare with, as I will explain below.
```suggestion
        self.inactive_top_comp_subdir = None
        if with_inactive_top_cells:
            self.inactive_top_comp_subdir = os.path.join(mesh_name, ic_dir, name)
            ic_dir = '{}_inactive_top'.format(ic_dir)
```
Also, make sure to add ``inactive_top_comp_subdir`` as an attribute in the docstring for the class above.